Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/EntryDelegate.cpp

```diff
@@ -148,6 +149,80 @@ namespace AzToolsFramework
             return m_iconSize;
         }
 
+        void SearchEntryDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option, const QModelIndex& index) const
+        {
+            auto data = index.data(AssetBrowserModel::Roles::EntryRole);
+            if (data.canConvert<const AssetBrowserEntry*>())
+            {
+                QStyleOptionViewItem opt(option);
+
+
```

Comment:
uber nit: extra line break here, usually we only have 1 max at a time in function blocks